src/Transactions/TransactionsList/TransactionsList.js

```diff
@@ -177,8 +173,8 @@ const TransactionsList = ({
       </ResultsPane>
 
       <Route
-        path={`${match.path}/:id/view`}
-        component={TransactionDetails}
+        path={`${match.path}/transaction/:id/view`}
+        render={() => <TransactionDetails baseUrl={match.url} />}
```

Comment:
why not to pass react-router props?

Reply:
`TransactionDetailsContainer` is also `withRouter`.